ArduPlane/mode_qrtl.cpp

```diff
@@ -40,20 +71,45 @@ void ModeQRTL::update()
  */
 void ModeQRTL::run()
 {
-    quadplane.vtol_position_controller();
-    if (poscontrol.get_state() > QuadPlane::QPOS_POSITION2) {
-        // change target altitude to home alt
-        plane.next_WP_loc.alt = plane.home.alt;
-    }
-    if (poscontrol.get_state() >= QuadPlane::QPOS_POSITION2) {
-        // start landing logic
-        quadplane.verify_vtol_land();
-    }
+    switch (submode) {
+        case SubMode::climb: {
+            quadplane.waypoint_controller();
```

Comment:
if we were in assist but flying fast (or in fwd transition) then I think this would result in us flying backwards to the point QRTL started? could be nasty

Reply:
I did not test this case, I guess just need to make sure that its set to the target to the stopping point. 

We could narrow the scope to just Qmodes and let assist behave as before.

Reply:
Now offsets to stopping point with:

```
plane.next_WP_loc.offset_bearing(degrees(plane.ahrs.groundspeed_vector().angle()), plane.quadplane.stopping_distance());
```

Reply:
great, still needs some testing in RealFlight with a few quadplanes. It will be running the full pos controller during that slowdown, we need to make sure that copes ok

Reply:
i've done some testing and seems good